src/prelude/prim.mo

```diff
@@ -478,3 +478,13 @@ func regionStoreBlob(r : Region, offset : Nat64, val :  Blob) : () =
 let call_raw = @call_raw;
 
 func performanceCounter(counter : Nat32) : Nat64 = (prim "performanceCounter" : (Nat32) -> Nat64) counter;
+
+// predicates for motoko-san
+
+func forall<T>(f: T -> Bool): Bool {
```

Comment:
I wonder what happens if these are called from compiled code rather that in viper mode. I expect the compiler will just crash complaining about an unknown Prim. I think we can live with this for now.

Note to self: possible fixes:
1) conditionally extend prim.mo in prelude.ml, the way we add timer support; or 
2) add a special deprecation error that triggers only in viper mode (as  for M0199 in https://github.com/dfinity/motoko/pull/4575)